terra-workspace-manager-client/build.gradle

```diff
@@ -66,35 +66,35 @@ dependencies {
     testImplementation group: 'org.hamcrest', name: 'hamcrest', version: '2.2'
 }
 
-// -- OpenAPI Client Generation --
-import org.openapitools.codegen.config.CodegenConfigurator
-import org.openapitools.codegen.DefaultGenerator
-def openapiSourceFile = "$projectDir/../src/main/resources/api/service_openapi.yaml"
-def openapiTargetFolder = 'src/main/java'
 
-task generateApi {
-    // Get the openapi file from the parent source directory
-    inputs.file("$openapiSourceFile")
-    outputs.dir("$projectDir/$openapiTargetFolder")
-    doLast {
-        def config = new CodegenConfigurator()
-        config.setInputSpec("file:///$openapiSourceFile")
-        config.setOutputDir("$projectDir/build")
-        config.setGeneratorName("java")
-        config.setAdditionalProperties([
-            'interfaceOnly' : 'true',                  // only generate the model and API interfaces
-            'library'       : 'jersey2',               // do jersey code generation so we use Jackson JSON
-            'dateLibrary'   : 'java8',                 // do java8 code generation
-            'apiPackage'    : 'bio.terra.workspace.api',
-            'modelPackage'  : 'bio.terra.workspace.model',
-            'invokerPackage': 'bio.terra.workspace.client',
-            'sourceFolder'  : openapiTargetFolder
-        ])
-        new DefaultGenerator().opts(config.toClientOptInput()).generate()
+openApiGenerate {
```

Comment:
does the "Publishing" portion of the README need to be updated with this change?